Carbon.IIS/Functions/Enable-CIisSsl.ps1

```diff
@@ -113,7 +118,9 @@ function Enable-CIisSsl
         $intFlag = $intFlag -bor $SslFlags_Ssl128
     }
 
-    $section = Get-CIisConfigurationSection -SiteName $SiteName -VirtualPath $VirtualPath -SectionPath 'system.webServer/security/access'
+    $sectionPath = 'system.webServer/security/access'
+    $section =

```

Comment:
Line formatting, perhaps

Reply:
Our standard is all on one line if possible (no line should be longer than 120 characters). If too long, put variable name and operator on one line, and the value on another line indented. If that is still more than 120 characters, then if the value is being returned from a cmdlet/function, put everything on the same line, except all arguments after the first. Each argument should be on its own line, indented to the same column as the name of the first argument.

In this case, able to stay under 120 characters just be moving variable value expression to the next line.